java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2245,6 +2247,33 @@ public T lastsave() {
         return getThis();
     }
 
+    /**
+     * Deletes all the keys of all the existing databases. This command never fails.<br>
+     * To explicitly specify the flushing mode, use {@link #flushall(FlushMode)}.
+     *
+     * @see <a href="https://redis.io/commands/flushall/">redis.io</a> for details.
+     * @return Command Response - <code>OK</code> to confirm that the databases were successfully
```

Comment:
The docs in "ServerManagementCommands/ServerManagementClusterCommands" don't contain the "to confirm that..." part, do we want to add it there to be consistent or doesn't really matter?